environment.yml

```diff
@@ -98,7 +98,7 @@ dependencies:
 
   - pyqt>=5.9.2  # pandas.read_clipboard
   - pytables>=3.4.3  # pandas.read_hdf, DataFrame.to_hdf
-  - s3fs  # pandas.read_csv... when using 's3://...' path
+  - s3fs>=0.4.0  # pandas.read_csv... when using 's3://...' path (also brings in fsspec)
```

Comment:
Can you also include fsspec at it's minimum supported version here?

Reply:
Also add to `pandas.compat._optional` and  the table in `doc/source/install.rst`.